datalad/support/exceptions.py

```diff
@@ -330,7 +332,7 @@ def __init__(self, results=None, failed=None, msg=None):
     def __str__(self):
         super_str = super(IncompleteResultsError, self).__str__()
         return "{} {}" \
```

Comment:
May be then also make it `"{}{}{}".format(super_str, os.linesep, pformat(self.failed))` at least?
Still separating both by a space seems a bit weird if it's not a one-line output anymore. In your example output I find the `None [ ...` completely unclear wrt what it's trying to tell me.

Reply:
Re None - note that I am only changing formatting of failed, and that None is the absent message given by super. I am only trying to make exception message more human parsable, not absolve all of its sins ;-)

Reply:
Yes, I'm aware. What I'm saying is, that if we generate mutliline output anyway, that it's weird to have that one separated by a space only, suggesting some connection to what follows in that line.

Reply:
Yeap yeap - I am putting my pants on already and will go downstairs to my laptop to tune it all up. I agree, it would be useful to provide better context of we are to improve it, but then just a new line probably wouldn't suffice